automation/jleechanorg_pr_automation/tests/test_minimax_preflight_env.py

```
@@ -0,0 +1,124 @@
+
+        # Get the env passed to validate_cli_two_phase
+        call_kwargs = mock_validate.call_args.kwargs
+        env = call_kwargs.get("env", {})
+
+        # CRITICAL: Verify ANTHROPIC_API_KEY is set from MINIMAX_API_KEY
+        self.assertIn(
+            "ANTHROPIC_API_KEY",
+            env,
+            "ANTHROPIC_API_KEY must be in validation env for MiniMax"
+        )
+        self.assertEqual(
```

Ruff (0.15.0)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)